### ISSUE TITLE
Game fails to load fully due to critical error with Create and the Create Connected addon.

### ISSUE THREAD

**Issue by Timtee** [hlysine/create_connected#2]
[crash-2023-12-16_17.12.19-fml.txt](https://github.com/hlysine/create_connected/files/13694193/crash-2023-12-16_17.12.19-fml.txt)
Create seemingly ceases to just stop functioning and stops the game from loading further when the Create Connected mod is active.
Note, I do not know how you have your mod integrate with Create, or even what version of Create it is meant to run alongside with... but clearly it causes instabilities where an injection check of at least one of the mixin files completely fails, causing Create itself to critically error out.
Also the reason I bring this report to you instead of the Create devs is because I know this error is caused by this mod alone.

Hopefully with this log you can fix whatever within the mixins is causing Create to basically have a stroke simply put.

**Comment by hlysine** [comment]
Does this version fix your issue?
[UNZIP ME create_connected-0.0.2-dev-mc1.20.1.zip](https://github.com/hlysine/create_connected/files/13694638/UNZIP.ME.create_connected-0.0.2-dev-mc1.20.1.zip)

The mod is targeting Create 0.5.1e, but I don't see any reason why it wouldn't work with 0.5.1f

**Comment by Timtee** [comment]
To clarify, changing versions to that version when I first experienced this issue. Did not work. Because naturally my first response was to see about downgrading to the prior version of Create and see if that fixed it. It did not.

**Comment by Timtee** [comment]
> Does this version fix your issue? [UNZIP ME create_connected-0.0.2-dev-mc1.20.1.zip](https://github.com/hlysine/create_connected/files/13694638/UNZIP.ME.create_connected-0.0.2-dev-mc1.20.1.zip)
> 
> The mod is targeting Create 0.5.1e, but I don't see any reason why it wouldn't work with 0.5.1f

[crash-2023-12-18_15.44.03-fml.txt](https://github.com/hlysine/create_connected/files/13708842/crash-2023-12-18_15.44.03-fml.txt)
I honestly did not even notice you link a dev version but... yeah it didn't help. Also the above log is with 0.5.1.f while the one below is 0.5.1.e
[crash-2023-12-18_15.50.42-fml.txt](https://github.com/hlysine/create_connected/files/13708875/crash-2023-12-18_15.50.42-fml.txt)
Basically, not even the dev variant helped.

**Comment by hlysine** [comment]
Are you using a fork of Create by any chance? I can't reproduce the crash in any way, and the inject that causes the crash should work in most recent versions of Create.

**Comment by Timtee** [comment]
> Are you using a fork of Create by any chance? I can't reproduce the crash in any way, and the inject that causes the crash should work in most recent versions of Create.

As far as I'm aware, no. I'm just using standard Forge Create, which is honestly the only version from the main dev team that doesn't immediately throw up because of fly wheel having bad interactions with graphics solutions similar to Sodium that are available for Forge.
Though I am running Create alongside all of the addons for Create that are stable and functional. (there are a few that don't function. Though if you look at the mod list within the logs, you should be able to see which of the addons for Create I've got.)

**Comment by hlysine** [comment]
It turns out this mod crashes outside of development environments because the inject is not remapped. That was quite embarrassing... Anyway, v0.0.4 should fix your issue.
